mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -859,8 +867,19 @@ class WidgetAdapter(
             }
         }
 
-        override fun onClick(view: View) {
-            connection.httpClient.sendItemCommand(boundWidget?.item, view.tag as String)
+        override fun onTouch(view: View, event: MotionEvent): Boolean {
```

Comment:
I think I'd prefer keeping `onClick` if there's no release command, since reacting to pointer down events directly is somewhat prone to mis-clicks. I understand it's not possible to do otherwise with release commands, but I imagine those to stay a niche use case.

Reply:
I'm still trying to figure out this part. Even if we use onclick when no release command, I'd still have to implement click+release so they work properly.

Reply:
> I'd still have to implement click+release so they work properly.

What do you mean by that?

AFAICT, the logic could be (pseudo code):
```
if (has release command) {
    setOnClickListener(null)
    setOnTouchListener(this)
} else {
    setOnClickListener(this)
    setOnTouchListener(null)
}
````

The `onClick` implementation could stay as it is now, and you'd just *additionally* add the touch listener.

Reply:
It's a bit complicated. When handling the ontouch event, even after calling `view.performClick()` as per [the warning](https://googlesamples.github.io/android-custom-lint-rules/checks/ClickableViewAccessibility.md.html), the standard accesibility action (haptic feedback) didn't occur. 

I implemented a workaround that makes both with and without release command to work using a combination of both onclick and ontouch handlers.

In the case of no releasecommand, it still goes through onclick as normal, even though ontouch is also implemented. It's probably easier to understand what I'm trying to say by to looking at the code in the next commit.